.github/workflows/deploy_staging.yml

```
@@ -0,0 +1,63 @@
+name: Deploy Mapping Viz Staging
+
+on:
+    # Run this workflow on push to master (incl PR merges)
+    push:
+        branches: [ master ]
+
+    # Allow running this workflow manually from the Actions tab
+    workflow_dispatch:
+
+jobs:
+  build_staging:
+    name: Build Staging
+    runs-on: ubuntu-latest
+    env:
+      HEAD_COMMIT: ${{ github.sha }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: azure/login@v1
+      with:
+          creds: ${{ secrets.AZURE_STATIC_SITES }}
+
+    - name: Node.js build
+      uses: actions/setup-node@v1
+      with:
+        node-version: '16.x'
+    - run: yarn install --frozen-lockfile
+    - run: yarn build
+      env:
+        REACT_APP_ENV: development
```

I think this will be less confusing but it needs double checking to make sure it works. According to the PJC config file, `staging` is a valid value for the API client environment. 
```suggestion
        REACT_APP_ENV: staging
```